src/js/tabs/pid_tuning.js

```diff
@@ -13,12 +13,28 @@ TABS.pid_tuning = {
 };
 
 TABS.pid_tuning.initialize = function (callback) {
+
     var self = this;
 
     if (GUI.active_tab !== 'pid_tuning') {
         GUI.active_tab = 'pid_tuning';
     }
 
+    // Update filtering defaults based on API version
+    var FILTER_DEFAULT = DEFAULT
+    if (semver.eq(CONFIG.apiVersion, "1.40.0")) {
+        FILTER_DEFAULT.dterm_lowpass2_hz = 200;
+    } else if (semver.gte(CONFIG.apiVersion, "1.41.0")) {
+        FILTER_DEFAULT.gyro_lowpass_hz = 150;
+        FILTER_DEFAULT.gyro_lowpass_type = 1;   // BIQUAD
+        FILTER_DEFAULT.gyro_lowpass2_hz = 0;
+        FILTER_DEFAULT.gyro_lowpass2_type = 1;  // BIQUAD
+        FILTER_DEFAULT.dterm_lowpass_hz = 150;
+        FILTER_DEFAULT.dterm_lowpass_type = 1;  // BIQUAD
+        FILTER_DEFAULT.dterm_lowpass2_hz = 150;
+        FILTER_DEFAULT.dterm_lowpass2_type = 1;  // BIQUAD
+    }
```

Comment:
Not sure if this is the best approach. We have defaults defined in the fc.js file (I think remember) that are modified here. I think it's better to have them in only one place, two options:
- Change in the fc.js the defaults by a function that returns them based on the version.
- Move the defaults here. I think we only have PID and filter defaults if I'm not wrong in all the Configurator.
What do you think?

Reply:
I'm going to be out until late tonight, but I'll look at adding a `getFilterDefaults()` function to fc.js and use that to return the values based on version. Then all the maintenance will be in fc.js.